index.bs

```diff
@@ -795,6 +800,56 @@ To <dfn>initialize WebTransport over HTTP</dfn>, given a {{WebTransport}} object
 
 </div>
 
+<div algorithm="compute a certificate fingerprint">
+To <dfn>compute a certificate fingerprint</dfn>, do the following:
+1. Let |cert| be the input certificate, represented as a DER encoding of
+   Certificate message defined in [[!RFC5280]].
+1. Compute the SHA-256 hash of |cert|.  Format it as `fingerprint` BNF
+   rule described in Section 5 of [[!RFC8122]].
+
+</div>
+
+<div algorithm="verify a certificate fingerprint">
+To <dfn>verify a certificate fingerprint</dfn>, do the following:
+1. Let |fingerprints| be the input array of fingerprints.
+1. Let |referenceFingerprint| be the [=compute a certificate fingerprint|computed
+   fingerprint=] of the input certificate.
+1. For every fingerprint |fingerprint| in |fingerprints|:
+   1. If {{RTCDtlsFingerprint/algorithm}} of |fingerprint| is equal to "sha-256",
+      and {{RTCDtlsFingerprint/value}} of |fingerprint| is equal to
+      |referenceFingerprint|, the certificate is valid.  Return true.
+1. Return false.
+
+</div>
+
+<div algorithm="cleanup a WebTransport">
+To <dfn for="WebTransport">cleanup</dfn> a {{WebTransport}} |transport| with |reason|,
+|error| and a boolean |abruptly|, run these steps:
```

Comment:
Nit: To me, this sounds like the "close" algorithm more than a "cleanup" algorithm (it sets the `"closed"` state and settles the `closed` promise). I'd rename it to that or — since it's only called from one place — maybe inline it? "Abruptly" makes more sense then too (an abrupt close makes more sense than an abrupt cleanup).

Reply:
As in the PR description I'm planning to use it in the server-initiated termination case and the network error case.

I'd like to avoid calling it "close" because it's used by the server initiated termination and the network error case. Calling it "close" will give an wrong impression that it's for WebTransport.close.